### ISSUE TITLE
Rust turbofish uses unexpected angle bracket/colon ligature

### ISSUE THREAD

**Issue by ian-h-chamberlain** [githubnext/monaspace#171]
Rust code may contain a "turbofish", which is written like so:
```rs
let v = Vec::<i32>::new();
```

Since there is a ligature for `>:`, this seems to take priority rather than the second `::`. This can also have a side effect of making syntax highlighting look a bit weird if the `>` is a different token type than `::`, like so:

![Screenshot 2023-12-05 at 10 20 24](https://github.com/githubnext/monaspace/assets/11131775/694c865b-e9ba-463c-be78-6a0c526a4381)

Both of these ligatures are part of `calt` so as far as I can tell there's no way to disable just `>:` with configuration.

**Comment by hikionori** [comment]
Maybe this can help you?
```json
"editor.fontLigatures": "'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08', 'dlig'",
````

**Comment by ian-h-chamberlain** [comment]
That would probably work, but also has the side effect of disabling all the other `calt` ligatures, which I would want in Rust, e.g. `&&`, `//`, `///`, `..`, `?.`, `::`, etc.

The tradeoff of having all those other ligatures is probably worth it vs this one corner case, but I think the optimal solution would just be to give the `::` ligature precedence over `>:` (if that's possible), or define a new contextual alternate for `>::`, or something along those lines. 

Another alternative would be to move `>:` and related ligatures into their own stylistic set (`ss09` or something), but I'm not sure that would really match up with how the rest of the stylistic sets are structured or what they are meant to be used for.

**Comment by hikionori** [comment]
I honestly don't see a big problem with this (turning off calt) simply because `//` and other is not that different in my opinion.

**Comment by hikionori** [comment]
Okay, yes I agree, I’m wrong. And in general, according to the development idea, they should have foreseen the turbofish syntax and how it should be displayed.

**Comment by hikionori** [comment]
Okay, yes I agree, I’m wrong. And in general, according to the development idea, they should have foreseen the turbofish syntax and how it should be displayed.

**Comment by hikionori** [comment]
Hey, mate. I tested it in my VSCode and everything works correctly.
![image](https://github.com/githubnext/monaspace/assets/66259055/4723a021-c643-4572-9f19-183f60635670)

**Comment by idan** [comment]
Fixed in ca8015b27bdc6d8e1201b1ad722f4ae3c38970d0
